.github/workflows/cron_nightly.yml

```
@@ -10,19 +10,27 @@ jobs:
   nightly:
     runs-on: ubuntu-latest
     name: Nightly
+    strategy:
+      matrix:
+        include:
+          - branch: develop
+            instance: develop
+          - branch: 1.7.8.x
+            instance: release
```

```suggestion
      matrix:
        include:
          - branch: develop
            instance: develop
          - branch: 1.7.8.x
            instance: release
      fail-fast: false
```

This should prevent skipping one job if the other one fails.